Xamarin.Forms.Platform.Android/Renderers/PageRenderer.cs

```diff
@@ -115,5 +123,57 @@ void UpdateBackgroundImage(Page view)
 			if (!string.IsNullOrEmpty(view.BackgroundImage))
 				this.SetBackground(Context.GetDrawable(view.BackgroundImage));
 		}
+
+		void IOrderedTraversalController.UpdateTraversalOrder()
+		{
+			// traversal order wasn't added until API 22
+			if ((int)Build.VERSION.SdkInt < 22)
```

Comment:
Could we use the Compat library?
https://developer.android.com/reference/android/support/v4/view/accessibility/AccessibilityNodeInfoCompat

```C#
global::Android.Support.V4.View.Accessibility.AccessibilityNodeInfoCompat.Wrap(control.CreateAccessibilityNodeInfo())
						.SetTraversalAfter(thisControl);
```

Reply:
Could we use the Compat library?
https://developer.android.com/reference/android/support/v4/view/accessibility/AccessibilityNodeInfoCompat

```C#
global::Android.Support.V4.View.Accessibility.AccessibilityNodeInfoCompat.Wrap(control.CreateAccessibilityNodeInfo())
						.SetTraversalAfter(thisControl);
```